v2/community/sdks.mdx

```diff
@@ -27,14 +27,14 @@ See the [supertokens-auth-react reference documentation](/docs/auth-react/instal
 
 If you want to only implement session management with SuperTokens, then you can use any frontend framework (React, Vue, Angular etc..) along with our [supertokens-website](/docs/website/installation) SDK.
 
-### Difference between the `auth-react` and `supertokens-website` sdks:
+## Difference between the `supertokens-auth-react` and `supertokens-website` sdks:
```

Comment:
The placement of this entire section is off.. makes no sense to put it here. Please think where it would be better,